Test/ServerTelemetryChannel.Test/Shared.Tests/Implementation/TelemetryBufferTest.cs

```diff
@@ -112,6 +116,31 @@ public void AddsTelemetryToBufferUntilItReachesMax()
             }
 
             [TestMethod]
+            public void TelemetryBufferDoNotGrowBeyondMaxBacklogSize()
```

Comment:
Are there actually two copies of this method/file or is the code review diff confused?